okhttp/src/test/java/io/grpc/okhttp/OkHttpServerTransportTest.java

```diff
@@ -1215,11 +1275,16 @@ private void handshake(Settings settings) throws Exception {
   }
 
   private static Buffer createMessageFrame(String stringMessage) {
+    return createMessageFrame(stringMessage, 0);
+  }
+
+  private static Buffer createMessageFrame(String stringMessage, int paddingLength) {
```

Comment:
I don't see the point of this new method, and it is a bit misleading. This seems to only work because nothing consumes the padding. When outside of the test, the buffer would have paddingLength-1 extra data, because 1 byte was consumed to communicate the padding length.

It seems this could just be removed and use the old non-padding-enabled creation.